### DISCUSSION TITLE
How can I make a child item draggable to different parents elements?

### DISCUSSION THREAD

**Discussion by diegomaldonadov** [clauderic/dnd-kit#963] [General]
trying to make child elements droppable to other parents, can see the code here.

https://codesandbox.io/s/sad-hill-83nqoo?file=/src/App.js

**Comment by mattbal** [comment]
From examples docs: https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-multiple-containers--basic-setup

Code: https://github.com/clauderic/dnd-kit/blob/master/stories/2%20-%20Presets/Sortable/MultipleContainers.tsx
